docs/en/operations/system-tables.md

```diff
@@ -1335,4 +1335,95 @@ Columns:
 
 If the storage policy contains more then one volume, then information for each volume is stored in the individual row of the table.
 
+## system.quotas {#system_tables-quotas}
+Contains information about [quotas](quotas.md).
+
+Columns:
+-  `name` ([String](../sql-reference/data-types/string.md)) — Quota name.
+-   `id` ([UUID](../sql-reference/data-types/uuid.md)) — Quota ID.
+-   `storage`([String](../sql-reference/data-types/string.md)) — Storage of quotas. Possible value: "users.xml" if a quota configured in the users.xml file, "disk" if a quota configured by an SQL-query.
+-   `key_type` ([Enum8](../sql-reference/data-types/enum.md)) — The type of key that the quota is tracked for. Values:
```

Comment:
I've slightly changed that. `key_type` is now an array of Enum8. `none` has been changed to `[]`, `client key or user name` => `[client_key, user_name]`.

Reply:
I see that another rows in this table changed too?

Reply:
*columns, of course))